src/css/blocks/page.css

```diff
@@ -0,0 +1,8 @@
+.page {
+  color: #060b35;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 10px;
+  font-weight: 500;
+}
```

Comment:
it's overkill ;) you can move this style ti product card

Reply:
You do not have block `page` in your html, whom is these styles for?